editoast/src/views/infra/mod.rs

```diff
@@ -489,7 +551,17 @@ async fn cache_status(
         infra: payload.infra,
     };
     let response = infra_request.fetch(core.as_ref()).await?;
-    Ok(Json(response))
+    Ok(response)
+}
+
+#[get("/cache_status")]
+async fn cache_status(
+    payload: Option<Json<StatePayload>>,
+    db_pool: Data<DbPool>,
+    core: Data<CoreClient>,
+) -> Result<Json<HashMap<String, InfraStateResponse>>> {
+    let payload = payload.unwrap_or(Json(StatePayload { infra: None }));
```

Comment:
I think that using `Either` here is more explicit since actix provides it. Apparently actix maps any error to `None` when an `Option<>` is given but I'm not convinced we want to rely on that kind of behavior. 

```suggestion
    payload: Either<Json<StatePayload>, ()>,
    db_pool: Data<DbPool>,
    core: Data<CoreClient>,
) -> Result<Json<HashMap<String, InfraStateResponse>>> {
    let payload = match payload {
        Either::Left(state) => state.into_inner(),
        Either::Right(_) => Default::default(),
    };
```